### PR TITLE
NeoDropdown issue

### LINKED ISSUE #7520: NeoDropdown issue
when browsing the app I've noticed this error several times in the console

```log
Uncaught TypeError: can't access property "getBoundingClientRect", ele is null
```

comming from 
https://github.com/kodadot/nft-gallery/blob/fcf63b4d9655c088c5e428bbbc58ff5f71cb9076/libs/ui/src/components/NeoDropdown/NeoDropdown.vue#L95-L98

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix
  - [x] fix eslint warning 

  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #7520
  - [ ] Requires deployment <snek/rubick/worker>

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [ ] My fix has changed UI

  ## Copilot Summary
  
### <samp>🤖 Generated by Copilot at 01fbc06</samp>

Removed unused imports and computed properties from `GalleryItemBuy.vue` and `ItemsGridImage.vue` to optimize performance and readability. Fixed potential errors and formatting issues in `NeoDropdown.vue` and `NeoNftCard.vue`.

  
### <samp>🤖 Generated by Copilot at 01fbc06</samp>

> _We're the masters of the code, we purge the unused imports_
> _We use optional chaining, to avoid the null aborts_
> _We follow the style rules, we add the commas where they're due_
> _We're the guardians of the grid, we make the images shine through_

### CODE REVIEW

**Comment by prury** [comment]
i'm not being able to reproduce this one

File: components/items/ItemsGrid/ItemsGridImage.vue [hunk 1]
```diff
@@ -117,9 +117,6 @@ const nftStack = computed(() =>
 const isAvailbleToBuy = computed(() =>
   nftStack.value.some((nft) => Number(nft.price) > 0),
 )
-const anyAvailableForListing = computed(() =>
-  nftStack.value.some((nft) => !Number(nft.price)),
-)
```
Comment (roiLeo) [inline components/items/ItemsGrid/ItemsGridImage.vue:122]: why remove it's still used in this component

Comment (Jarsen136) [reply components/items/ItemsGrid/ItemsGridImage.vue:122]: If you check the latest nuxt branch, you will found that it's not used anymore.

**Comment by Jarsen136** [comment]
> i'm not being able to reproduce this one

How to reproduce this issue ?

1. open `https://nuxt.kodadot.xyz/`
2. enter the explorer page
3. hover over some of the menu and see the dropdown
4. back to landing page
5. adjust the width of window screen 
6. you will found there are some error on the `console`

**Review by roiLeo — APPROVED** [review]

**Pull request merged**